proto/query.proto

```diff
@@ -322,6 +322,8 @@ message ExecuteOptions {
   // if the user has created temp tables, Vitess will not reuse plans created for this session in other sessions.
   // The current session can still use other sessions cached plans.
   bool has_created_temp_tables = 12;
+
+  string gtid_set = 13;
```

Comment:
This field name seems too non-specific. Which gtid_set is this supposed to be?

Reply:
After offline discussion and thinking more about it. I think it would be better to call it position and use the flavour interface to interactive with this position.